.env.sample

```diff
@@ -0,0 +1,3 @@
+LD_CONTAINER_NAME=linkding
```

Comment:
Can you please add `.env` to `.gitignore`? Would make it a bit safer IMO, since the file should not be under version control.

Reply:
Nevermind, it already is. Took my IDE a while to update.

Reply:
`.env` is already excluded

```
# dotenv environment variables file
.env
```